build-tools/check-boot-times/Program.cs

```diff
@@ -397,6 +401,20 @@ static Task<bool> RunProcess (string filename, string arguments, int timeout, Fu
 					process.StartInfo.RedirectStandardError = true;
 					process.EnableRaisingEvents = true;
 
+					var sdkRootKey = "ANDROID_SDK_ROOT";
+					var prefsRootKey = "ANDROID_PREFS_ROOT";
+					if (process.StartInfo.EnvironmentVariables.ContainsKey (sdkRootKey))
+						process.StartInfo.EnvironmentVariables [sdkRootKey] = sdkPath;
+					else
+						process.StartInfo.EnvironmentVariables.Add (sdkRootKey, sdkPath);
+
+					if (!string.IsNullOrWhiteSpace (avdHome)) {
+						if (process.StartInfo.EnvironmentVariables.ContainsKey (prefsRootKey))
```

Comment:
This check isn't necessary; you can just always set it, even if it doesn't already exist:

```csharp
process.StartInfo.EnvironmentVariables [prefsRootKey] = avdHome;
```